### PR TITLE
Reduce number of pruning iterations

### PR DESCRIPTION
**Pull request by NRauschmayr**
*Issue #, if available:*
Reducing the number of pruning iterations to minimize the runtime. https://github.com/aws/amazon-sagemaker-examples/pull/2145#issuecomment-819934990

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: sagemaker-debugger/pytorch_iterative_model_pruning/iterative_model_pruning_alexnet.ipynb [hunk 1]
```diff
@@ -567,7 +567,7 @@
    "source": [
     "### Run iterative model pruning\n",
     "\n",
-    "After having gone through the code step by step, we are ready to run the full worfklow. The following cell runs 10 pruning iterations: in each iteration of the pruning a new SageMaker training job is started, where it emits gradients and activation outputs to Amazon S3. Once the job has finished, filter ranks are computed and the 100 smallest filters are removed.\n",
+    "After having gone through the code step by step, we are ready to run the full worfklow. The following cell runs 1 pruning iteration: in each iteration a new SageMaker training job is started, where it emits gradients and activation outputs to Amazon S3. Once the job has finished, filter ranks are computed and the 100 smallest filters are removed.\n",
```
Comment (mchoi8739) [inline sagemaker-debugger/pytorch_iterative_model_pruning/iterative_model_pruning_alexnet.ipynb:570]: The following cell runs 1 pruning iteration for a tutorial purpose. Change the range of the for loop to 10 to replicate the same result shown in the [Pruning machine learning models with Amazon SageMaker Debugger and Amazon SageMaker Experiments](https://aws.amazon.com/blogs/machine-learning/pruning-machine-learning-models-with-amazon-sagemaker-debugger-and-amazon-sagemaker-experiments/) blog and the figure below the cell. In each iteration, SageMaker starts a new training job with a pruned model, and Debugger collects gradients and activation outputs to Amazon S3.

File: sagemaker-debugger/pytorch_iterative_model_pruning/iterative_model_pruning_resnet.ipynb [hunk 2]
```diff
@@ -524,7 +524,7 @@
    "source": [
     "### Run iterative model pruning\n",
     "\n",
-    "After having gone through the code step by step, we are ready to run the full worfklow. The following cell runs 10 pruning iterations: in each iteration of the pruning a new SageMaker training job is started, where it emits gradients and activation outputs to Amazon S3. Once the job has finished, filter ranks are computed and the 100 smallest filters are removed.\n",
+    "After having gone through the code step by step, we are ready to run the full worfklow. The following cell runs 1 pruning iteration: in each iteration a new SageMaker training job is started, where it emits gradients and activation outputs to Amazon S3. Once the job has finished, filter ranks are computed and the 100 smallest filters are removed.\n",
```
Comment (mchoi8739) [inline sagemaker-debugger/pytorch_iterative_model_pruning/iterative_model_pruning_resnet.ipynb:527]: The following cell runs 1 pruning iteration for a tutorial purpose. Change the range of the for loop to 10 to replicate the same result shown in the [Pruning machine learning models with Amazon SageMaker Debugger and Amazon SageMaker Experiments](https://aws.amazon.com/blogs/machine-learning/pruning-machine-learning-models-with-amazon-sagemaker-debugger-and-amazon-sagemaker-experiments/) blog and the figure below the cell. In each iteration, SageMaker starts a new training job with a pruned model, and Debugger collects gradients and activation outputs to Amazon S3.

**Review by mchoi8739 — APPROVED** [review]
